loadtest/store/memstore/store.go

```diff
@@ -101,6 +101,16 @@ func (s *MemStore) ChannelPosts(channelId string) ([]*model.Post, error) {
 	return channelPosts, nil
 }
 
+func (s *MemStore) PostsSince(ts int64) ([]model.Post, error) {
+	var posts []model.Post
+	for _, post := range s.posts {
+		if post.CreateAt > ts {
```

Comment:
Greater or greater and equal?

Reply:
I think the MM API uses greater but it's not that important. This is more of a utility function that I used to avoid picking posts completely at random.